web/views.py

```
@@ -7103,3 +7128,112 @@ def users_list(request: HttpRequest) -> HttpResponse:
+    if request.user.is_authenticated:
+        try:
+            vote = ForumVote.objects.get(topic=topic, user=request.user)
+            user_topic_vote = vote.vote_type
+        except ForumVote.DoesNotExist:
+            pass
+
+    # Get user votes on replies
+    user_reply_votes = {}
+    if request.user.is_authenticated:
+        reply_votes = ForumVote.objects.filter(reply__topic=topic, user=request.user).values_list(
+            "reply_id", "vote_type"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)